cli/crates/cli/src/create.rs

```diff
@@ -23,6 +23,10 @@ pub struct CreateArguments<'a> {
 
 #[tokio::main]
 pub async fn create(arguments: &Option<CreateArguments<'_>>) -> Result<(), CliError> {
+    create_impl(arguments).await
+}
+
+pub async fn create_impl(arguments: &Option<CreateArguments<'_>>) -> Result<(), CliError> {
```

Comment:
At some point this might become a shared runtime, although that'd need further changes to the CLI